front/src/layout/styles/themes.ts

```diff
@@ -16,6 +16,10 @@ const commonTheme = {
   fontFamily: 'Inter, sans-serif',
 
   spacing: (multiplicator: number) => `${multiplicator * 4}px`,
+
+  table: {
+    sideMarginInPx: 8,
```

Comment:
to be consistent with spacing above + I don't like the "side" name, I would do
horizontalCellMargin: 8px